backend/tests/controllers/authController.test.js

```
@@ -0,0 +1,40 @@
+const httpMocks = require('node-mocks-http');
+const authService = require('../../src/services/authService');
+const { register } = require('../../src/controllers/authController');
+const { sendSuccessResponse, sendErrorResponse } = require('../../src/utils/responseHelper');
+const { UserAlreadyExistsError } = require('../../src/utils/errors');
+
+jest.mock('../../src/services/authService');
+jest.mock('../../src/utils/responseHelper');
+
+describe('authController.register', () => {
+  let req, res;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)